api/compaction_manager.cc

```diff
@@ -38,6 +39,18 @@ static future<json::json_return_type> get_cm_stats(http_context& ctx,
         return make_ready_future<json::json_return_type>(res);
     });
 }
+static std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash> sum_panding_tasks(std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash> a,
+        const std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash>& b) {
+    for (auto&& i : b) {
+        if (a.find(i.first) != a.end()) {
+            a[i.first] += i.second;
+        } else {
```

Comment:
The "if" isn't needed. a[i.first] will create and initialize the map entry if it doesn't exist.